tests/integration/frameworks/wordpress/test_wordpress_apply_filters.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*DESCRIPTION
+The agent should properly instrument Wordpress apply_filters hooks.
+*/
+
+/*SKIPIF
+<?php
+if (version_compare(PHP_VERSION, "5.6", "<")) {
+  die("skip: PHP < 5.6 argument unpacking not supported\n");
+}
+*/
+
+/*INI
+newrelic.framework = wordpress
+*/
+
+/*EXPECT
+add filter
+add filter
+add filter
+f: string1
+add filter
+h: string3
+g: string2
+*/
+
+/*EXPECT_METRICS
```

Comment:
Would this be a good place to use `EXPECT_METRICS_EXIST` added in #771 to make the test more focused. This test only cares about following metrics being created:
`Framework/WordPress/Hook/f`, `Framework/WordPress/Hook/g`, `Framework/WordPress/Hook/h`.

Reply:
This will not be done to limit number of conflicts when syncing oapi branch.